website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -178,7 +197,19 @@ The [`disable_compat_19`](/docs/agent/options#telemetry-disable_compat_1.9) tele
 In prior Consul versions (1.10.x through 1.11.x), the config defaulted to `false`. In 1.12.x it defaulted to `true`.
 If you were using this flag, you must remove it before upgrading.
 
-## Consul 1.12.0
+### Modify Vault Policy for Vault CA Provider
+
+If upgrading to Consul 1.13.2 or later,
```

Comment:
Note to reviewers: I wasn't sure how to best copy the same guidance into multiple places that it's needed. Currently, this PR is just referencing the 1.14 content from the 1.11-1.13 sections (to avoid duplicate content). I'll obviously need to modify that somewhat to merge into 1.13 immediately (not waiting for 1.14 docs release to get this important info out).